modules/apps/batch-planner/batch-planner-test/src/testFunctional/tests/dependencies/objectEntry_import_sample.csv

```diff
@@ -0,0 +1,2 @@
+keywords,externalReferenceCode,taxonomyCategoryIds,name
```

Comment:
What I am not sure if the csv file only contains fields not the data with a real object entry, please help check with developers, thanks.